tasks/html/hard/index.html

```diff
@@ -10,23 +10,59 @@
     </head>
     <body>
         <nav id="navbar">
-            <!-- TODO: Add navbar -->
+            <ul style='none'>
```

Comment:
Page looks good! There is just this small issue where the `style='none'` should be `style='list-style: none;'` but other than that everything looks okay. No need to fix that, i'm going to merge this.